frontend/public/locales/en/gcweb.json

```
@@ -29,7 +29,9 @@
     "menu-security-settings.href": "{{baseUri}}/en/security-settings",
     "menu-contact-us.text": "Contact Us",
     "menu-contact-us.href": "{{baseUri}}/en/contact-us",
-    "menu-sign-out.text": "Sign Out"
+    "menu-sign-out.text": "Sign Out",
+    "banner-alert": "Info",
+    "banner-desc": "This is a prototype environment. No data is kept."
```

Change the names to a more specific name.

```suggestion
    "prototype-banner": {
      "alert": "Info",
      "desc": "This is a prototype environment. No data is kept."
    }
```